JSDemos/Demos/Tabs/Selection/Angular/app/app.component.css

```diff
@@ -1,26 +1,57 @@
-::ng-deep .content {
-  text-align: justify;
-  margin-top: 25px;
+::ng-deep #center-content {
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-::ng-deep #longtabs {
-  margin-top: 20px;
+::ng-deep .select-box-container,
+.multiview-container {
+  padding: 16px;
 }
 
-::ng-deep #scrolledtabs {
-  margin-top: 20px;
+::ng-deep #demo-items-container {
+  width: 680px;
 }
 
-::ng-deep #tabs {
-  margin-top: 60px;
+::ng-deep .employee-info {
+  display: flex;
+  align-items: center;
 }
 
-::ng-deep .caption {
+::ng-deep .employee-photo {
+  height: 80px;
+  width: 80px;
+  border-radius: 50%;
+  border: 1px solid #ebebeb;
```

Comment:
It's better to use class '.dx-theme-border-color' to get correct border color according to a correspond theme